### ISSUE TITLE
Error for QtLocation/private/qgeojson_p.h

### ISSUE THREAD

**Issue by edips** [ec1oud/map#1]
Hi. When I run the app, it gives error:
```
fatal error: QtLocation/private/qgeojson_p.h: No such file or directory
 #include <QtLocation/private/qgeojson_p.h>
```
I looked QtLocation/private/ folder, there isn't qgeojson_p.h file for Qt 5.12

**Comment by ec1oud** [comment]
Yeah it depends on the GSoC work, which didn't get committed for 5.12:  https://codereview.qt-project.org/#/c/236253/

**Comment by edips** [comment]
Thank you, I will try to download it and build it soon.

**Comment by ec1oud** [comment]
It's still quite incomplete I think, so this is not in a working state, and won't be until that is done.

**Comment by edips** [comment]
Yes. There is Esri's ArcGIS Runtime for QML which it can import geojson and shape file to map canvas. But it is commercial, it can be used for development only. Another way is to use QGIS's map canvas, which can import so many vector and raster map data with QGIS map canvas. I tried to install QGIS QML map canvas, I couldn't install it but I will try again. [Here](https://github.com/lutraconsulting) is a QGIS map canvas example app source. And [here](https://github.com/lutraconsulting/qgis-quick-demo-app/issues/1) it is discussed how to install.
